shared/getDeviceOnDash/artik710.md

```diff
@@ -9,24 +9,24 @@ __Note:__ This resinOS will completely write over the existing eMMC.
 <!-- <img src="/img/artik5/artik5-dev-kit.png" width="80%"> -->
 If you selected wifi as your connectivity type, remember to attach the wifi antennas before booting the board.
 
-We can now insert the 5VDC power cable and flip the power switch labelled `SW700`, to the `on` position. 
-We should now have some glowing LEDs on the underside of the board indicating a sign of life. Next we need to press and hold the `SW400 POWER` push button for 1 or 2 seconds, 
+We can now insert the 5VDC power cable and flip the power switch labelled `SW700`, to the `on` position.
```

Comment:
@curcuz i think here you are commenting on the wrong file, it should be `kitra710.md`, i have address these thing there.